.env.example

```diff
@@ -1,7 +1,10 @@
-POSTGRES_HOST=database
-POSTGRES_DB=db
-POSTGRES_USER=io-onboarding-pa-api
-POSTGRES_PASSWORD=password
+NODE_ENV=development
+COMPOSE_PROJECT_NAME=io-onboarding
```

Comment:
This new env var seems to be unused, what is it needed for?

Reply:
https://docs.docker.com/compose/reference/envvars/